src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IObjectCreationExpression.cs

```diff
@@ -1623,7 +2360,75 @@ class C2 : IEnumerable<int>
                         InConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
                         OutConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
 ";
-            VerifyOperationTreeAndDiagnosticsForTest<InitializerExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics, parseOptions: ImplicitObjectCreationOptions);
+            VerifyOperationTreeAndDiagnosticsForTest<InitializerExpressionSyntax>(comp, expectedOperationTree, expectedDiagnostics);
+
+            var m = comp.SyntaxTrees[0].GetRoot().DescendantNodes().OfType<MethodDeclarationSyntax>().Single(m => m.Identifier.ValueText == "M");
+            VerifyFlowGraph(comp, m, @"
```

Comment:
>VerifyFlowGraph [](start = 12, length = 15)

It looks like all tests adjusted in the second commit are using target-typed ```new```. I am assuming regular ```new``` had the same issue. Please confirm.

Reply:
It would be good to add verification for a couple of interesting scenarios with a regular ```new```.

---
In reply to: [528940856](https://github.com/dotnet/roslyn/pull/49534#discussion_r528940856) [](ancestors = 528940856)

Reply:
It looks like it doesn't, because the existing code uses a syntax check rather than IOperation check, which I missed on my first pass. I'll update the syntax check instead, we already have coverage of these scenarios for regular object creation.

---
In reply to: [528943004](https://github.com/dotnet/roslyn/pull/49534#discussion_r528943004) [](ancestors = 528943004,528940856)